.gitignore

```diff
@@ -8,3 +8,5 @@ __pycache__/
 .idea
 .vscode/
 *.rock
+*.egg-info/
+dist/
```

Comment:
I think there is a missing newline here :)

Reply:
Added, thanks!